lib/adapter/freshheads/CopyFilesToBuildDirAdapter.ts

```diff
@@ -40,19 +40,14 @@ export default class CopyFilesToBuildDirAdapter implements Adapter {
         next: NextCallback
     ) {
         const patterns: CopyPattern[] = [];
-        const isProduction = builderConfig.env === 'production';
 
         if (this.config.images) {
             patterns.push({
                 from: {
                     glob: path.resolve(process.cwd(), 'src/images/**/*'),
                 },
                 context: process.cwd(),
-
-                // @todo check why thie prefix is app/ below, and if we can change this to something else
-                to: isProduction
-                    ? 'app/[path][name].[hash].[ext]'
-                    : 'app/[path][name].[ext]',
+                to: 'images/[name].[hash].[ext]',
```

Comment:
Waar ik wel benieuwd naar ben is of deze aanpassing mogelijk naamconflicten kan veroorzaken? Is de hash hetzelfde voor alle bestanden in de huidige build van Webpack? Als dat zo is, dan zouden bijvoorbeeld alle bestanden die hetzelfde heten, en in die in de bron wel naast elkaar kunnen bestaan omdat ze in verschillende mappen zitten, na het builden kunnen conflicteren. Snap je wat ik bedoel? Even een goede om te testen denk ik ;)